apps/omg_watcher_info/lib/omg_watcher_info/db/repo.ex

```diff
@@ -32,17 +32,29 @@ defmodule OMG.WatcherInfo.DB.Repo do
 
   def insert_all_chunked(_schema_or_source, [], _opts), do: :ok
 
+  # Note that while this function allows us to get around postgres' max parameters limit
```

Comment:
delete this comment pls and close the issue

Reply:
done